CONTRIBUTING.md

```diff
@@ -13,6 +13,38 @@ virtualenv -p python3 venv && \
   pip install --requirement dev_requirements.txt
 ```
 
+Furthermore it's necessary to install node packages for utilities such as _prettier_:
+
+```bash
+npm install
+```
+
+## Testing
+
+You can execute unit tests directly like this (just an example testcase):
+
+```bash
+python3 -m unittest ./restic/restic_test.py
+```
+
+or you can run the complete testsuite including coverage analysis using this script:
+
+```bash
+./dev-scripts/build
+```
+
+If you enabled GIT hooks this script will be executed with each commit.
```

Comment:
This mentions Git hooks, but the way this is currently ordered, Git hooks are later in the document. Can we reorder it so that this section is after the section on enabling Git hooks?

Also, can we keep the casing of "Git" consistent with the rest of the doc?